README.md

```diff
@@ -1 +1,40 @@
-# my-portfolio
\ No newline at end of file
+# My Portfolio 
+
+> A very simple portfolio 
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+**You can download or clone it in your computer. After that you can open it in any text editor that you like*
+
+
+### Prerequisites
+
+## Authors
```

Comment:
- [ ]  Well done adding a README to make a professional readme file; kindly customize the sections, and please fill many sections as you can.